ee/localserver/request-id.go

```diff
@@ -78,6 +79,7 @@ func (ls *localServer) requestIdHandlerFunc(w http.ResponseWriter, r *http.Reque
 	response := requestIdsResponse{
 		Nonce:     ulid.New(),
 		Timestamp: time.Now(),
+		Origin:    r.Header.Get("Origin"),
```

Comment:
This will be empty because of how we construct this request in `krypto-ec-middleware` -- we don't bring along any of the wrapping request's headers. I was able to include the origin by updating the instantiation in `(e *kryptoEcMiddleware) Wrap` like this --

```
newReq := &http.Request{
	Method: http.MethodPost,
	URL: &url.URL{
		Scheme: r.URL.Scheme,
		Host:   r.Host,
		Path:   cmdReq.Path,
	},
	Header: make(http.Header),
}
newReq.Header.Set("Origin", r.Header.Get("Origin"))
```

Reply:
nice